tests/Feature/WidgetFeatureTest.php

```diff
@@ -153,6 +153,7 @@ public function test_register_a_widget_with_invalid(): void {
 		// Register widgets.
 		do_action( 'widgets_init' );
 
-		$this->setExpectedIncorrectUsage( 'Alley\WP\Features\{closure}' );
+		// Until Mantle supports it.
+		$this->ignoreIncorrectUsage( '*Alley\WP\Features*' );
```

Comment:
Temporary while Mantle adds support for the wildcard feature to `setExpectedIncorrectUsage`, so that it works for PHP 8.3 and PHP 8.4.